### ISSUE TITLE
Add possibility to pass mysqldump parameters

### ISSUE THREAD

**Issue by kinafu** [rugk/borg-cron-helper#79]
For my nextcloud (backup) I need to pass an option (`--default-character-set=utf8mb4`) to the mysqldump command  (in order to properly backup strings containing unicode smileys).

Using our tool `databasedump.sh` you should be able to pass arguments or environment variables to mysqldump.

If you're curious the [nextcloud docs ](https://docs.nextcloud.com/server/15/admin_manual/configuration_database/mysql_4byte_support.html)gave me the hint to check, wether one's backup strategy will still work with its new way of storing data in mysql.

**Comment by rugk** [comment]
Good idea! Should be easy to implement given we already have a [`specialParams`](https://github.com/rugk/borg-cron-helper/blob/018617df249b2f8d0a4652363b80fee134ffbe22/tools/databasedump.sh#L87) variable.
